src/xtopcom/xchain_upgrade/src/xchain_upgrade_center.cpp

```diff
@@ -75,6 +81,14 @@ namespace top {
             return  xtop_chain_fork_config_center::is_forked(_fork_config.block_fork_point, target);
         }
 
+        bool xtop_chain_fork_config_center::is_tx_forked(uint64_t fire_timestamp) noexcept {
+            xchain_fork_config_t const & _fork_config = xtop_chain_fork_config_center::get_chain_fork_config();
+            auto clock = _fork_config.tx_v2_fork_point.value().point;
+            constexpr uint64_t TOP_BEGIN_GMTIME = 1573189200;
+            auto clock_time_stamp = clock * 10 + TOP_BEGIN_GMTIME;
```

Comment:
repeat code of TOP_BEGIN_GMTIME and clock to gmt

Reply:
repeat code can't be reused without changing original functions